frontend/smart-kickers-game/src/App.js

```diff
@@ -24,6 +30,45 @@ function App() {
     };
   }, []);
 
+  const ScorePrevious = (value) => {
```

Comment:
Why is it needed?

Reply:
To get the previous score from each team. Thanks to that we can avoid the situation when you subtract the goal and the useEffect will add it as the goal - using `prevWhiteScore` or `prevBlueScore` in the if state in useEffect we can delete the last scored goal from the array when the goal subtraction occurs.
To sum up we have to check if you press the plus or minus button.
Can we make it easier?

Reply:
We would stay with it, when we will move this **Game History** to the new view (using routers) we will think about this idea  and even get rid of the `goalsArray` (moving it to the Go server).
Is it ok to make it that way?